### PR TITLE
Allow pasting image from clipboard

### PR DESCRIPTION
**Pull request by felipebueno**
This is another nice feature I wanted for a long time :)

input-ezgif com-cut

### CODE REVIEW

**Comment by felipebueno** [comment]
Not sure how to fix this

      /home/runner/work/stacker.news/stacker.news/components/form.js:274:26: 'DataTransfer' is not defined. (no-undef)

I'll keep reseaching

**Comment by felipebueno** [comment]
Oh, I just saw I can run the linter locally 😅

image

File: components/form.js [hunk 1]
```diff
@@ -256,6 +256,30 @@ export function MarkdownInput ({ label, topLevel, groupClassName, onChange, onKe
...
+
+    const item = items[0]
+    if (item.type.indexOf('image') !== 0) {
+      return
+    }
+
+    const blob = item.getAsFile()
+    const file = new File([blob], 'image', { type: blob.type })
+    /* global DataTransfer */
+    const fileList = new DataTransfer()
```
Comment (huumn) [inline components/form.js:275]:
```suggestion
    const fileList = new window.DataTransfer()
```

**Comment by huumn** [comment]
Works well! I modified to allow pasting of multiple images

**Comment by SatsAllDay** [comment]
Cool, I have been trying to do this as a user for ages and always forget it isn't supported. Nice work!

**Pull request merged**